R/get_analysis_date.R

```diff
@@ -16,7 +16,25 @@
 #  You should have received a copy of the GNU General Public License
 #  along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
-#' Get the analysis date under multiple conditions
+
+#' Derive analysis date for interim/final analysis given multiple conditions.
+#'
+#' To obtain the analysis date, consider the following multiple conditions:
+#' Condition 1: The planned calendar time for analysis.
```

Comment:
This will not be nicely formatted in the final help file. This is what it looks like locally:

![image](https://github.com/Merck/simtrial/assets/1608317/738cbae7-11b1-4dd4-8088-86618b6bc9f8)

Reply:
I recommend either using `\describe{}`, which looks like the following:

```
#' To obtain the analysis date, consider the following multiple conditions:
#' \describe{
#'   \item{Condition 1:}{The planned calendar time for analysis.}
#'   \item{Condition 2:}{The targeted events, encompassing both overall population
#' and stratum-specific events.}
#' }
```

which would result in

![image](https://github.com/Merck/simtrial/assets/1608317/75de5545-c578-4444-939a-98c52e8f9237)

or a Markdown list

```
#' * Condition 3: The maximum time extension required to achieve the targeted events.
#' * Condition 4: The planned minimum time interval after the previous analysis.
#' * Condition 5: The minimum follow-up time needed to reach a certain number of
#' patients in enrollments.
```

which would render as

![image](https://github.com/Merck/simtrial/assets/1608317/74fe25d7-5180-41bd-bbb7-44af82dff9bb)

Reply:
Hi @jdblischak , thank you for reviewing the formatting! I have just made an additional commit. Please feel free to make any edits or suggest improvements to the content if you think there are any further areas that could benefit from refinement.

Reply:
Great suggestions